### PR TITLE
Add datachannels to munge-sdp demo

### LINKED ISSUE #625: Add data channels to munge-sdp demo
It would be useful for testing data channels specifically.

### PR DESCRIPTION
**Pull request by KaptenJansson**
Do we really need a checkbox to toggle dataChannels or is it fine to always include it (like it does in this PR)?

Fixes #625

### CODE REVIEW

**Comment by KaptenJansson** [comment]
I also fixed localStream.stop().

File: src/content/peerconnection/munge-sdp/index.html [hunk 1]
```diff
@@ -63,6 +63,8 @@
         <video autoplay muted></video>
         <h2>Offer SDP</h2>
         <textarea></textarea>
+        <h2>DataChannel send</h2>
+        <textarea id="dataChannelSend" disabled>Data sent </textarea>
```
Comment (fippo) [inline src/content/peerconnection/munge-sdp/index.html]: is the extra whitespace intentional here?

**Comment by fippo** [comment]
LGTM -- the way you update the remote value almost implements a realtime chat. In the most inefficient way possible :-)

But i wonder if we really want to merge the datachannel chat demo into this... well, we can remove it when it becomes a maintenance issue.

**Comment by KaptenJansson** [comment]
Haha, yeah well I just wanted some continuous data flowing, we can remove the send box and not let the user affect the data sent?

File: src/content/peerconnection/munge-sdp/index.html (hunk 1 above)
Comment (KaptenJansson) [inline src/content/peerconnection/munge-sdp/index.html]: No, it was lefter over when I had the counter on the same line (Data sent: ). Will remove.

**Comment by fippo** [comment]
to see the non-effect this has now? +1 on removing the box and just sending something in the background

**Comment by KaptenJansson** [comment]
Do you want both boxes gone or just the send one?

**Comment by fippo** [comment]
oh.. both actually. Even though leaving the receive box makes this testable from selenium... hm..!

**Comment by KaptenJansson** [comment]
Hmm, what about storing the data in a global variable instead? Or an invisible element not attach to the DOM?

**Comment by fippo** [comment]
global variable (just a count of the amount of data received) sounds easiest.

**Comment by KaptenJansson** [comment]
Fixed in #638

**Pull request merged**